landing/src/app.html

```diff
@@ -2,9 +2,23 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8" />
-		<meta name="description" content="" />
+		<meta name="description" content="A living guide to sensitivity analysis and diagnostic model evaluation techniques for confronting the computational and conceptual challenges of multi-model, transdisciplinary workflows." />
 		<link rel="icon" href="%svelte.assets%/favicon.png" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
+
+		<meta property="og:title" content="Addressing Uncertainty in MultiSector Dynamics Research" />
+		<meta property="og:type" content="website" />
+		<meta property="og:url" content="https://immm-sfa.github.io/msd_uncertainty_ebook/" />
+		<link rel="image_src" href="https://immm-sfa.github.io/msd_uncertainty_ebook/images/ebook_thumbnail_1.png" type="image/png" />
+		<meta property="og:image" content="https://immm-sfa.github.io/msd_uncertainty_ebook/images/ebook_thumbnail_1.png" />
+		<meta property="og:image:secure_url" content="https://immm-sfa.github.io/msd_uncertainty_ebook/images/ebook_thumbnail_1.png" />
+		<meta property="og:image:type" content="image/png" />
+		<meta property="og:description" content="A living guide to sensitivity analysis and diagnostic model evaluation techniques for confronting the computational and conceptual challenges of multi-model, transdisciplinary workflows." />
+		<meta property="og:site_name" content="Addressing Uncertainty in MultiSector Dynamics Research" />
+		<meta property="og:locale" content="en_US" />
+		<meta name="twitter:title" content="Addressing Uncertainty in MultiSector Dynamics Research">
+		<meta name="twitter:description" content="A living guide to sensitivity analysis and diagnostic model evaluation techniques for confronting the computational and conceptual challenges of multi-model, transdisciplinary workflows.">
```

Comment:
@thurber are these closed?

Reply:
see 19 and 20

Reply:
it is definitely silly of me to be inconsistent, but luckily meta tags can be left open or closed 😅